cmd/cluster/powervs/destroy.go

```diff
@@ -63,13 +77,13 @@ func DestroyCluster(ctx context.Context, o *core.DestroyOptions) error {
 		inputErrors = append(inputErrors, fmt.Errorf("infrastructure ID is required"))
 	}
 	if o.PowerVSPlatform.Region == "" {
```

Comment:
Upper case seems right to me here even though is an Errorf they are capital letter words regardless.

Reply:
To be in consistent with other Errorf messages, modified this, reverted the changes.